packages/email/email.js

```diff
@@ -218,8 +218,11 @@ Email.send = function (options) {
   if (!send) return;
 
   const customTransport = Email.customTransport;
-  const transport = customTransport ? false : getTransport();
-  if (transport) {
+  if (customTransport) {
+    const packageSettings = Meteor.settings.packages?.email || {};
+    customTransport({ packageSettings, ...options });
+  } else if (Meteor.isProduction) {
```

Comment:
Please avoid else's using return statements.

For example:
```js
if (x) {
  // doSomething();
  return;
}

if (y) {
  // doSomethingElse();
  return;
}

// doLastCase();
```

Reply:
Fixed